src/test/requests.test.js

```diff
@@ -8,10 +8,10 @@ chai.should();
 describe('Requests Test', () => {
   it('should be able to reject if line manager', done => {
     chai.request(server)
-      .patch('/api/requests/2/reject')
+      .patch('/api/requests/9/reject')
       .set(
         'token',
-        'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiZW1haWwiOiJ0cmV5M0BnbWFpbC5jb20iLCJpYXQiOjE1ODEwMDI3NjJ9.FGgqQWqOEYO6ovfvq_bM58pYXY0VG1-3Wm-hQ59_koE'
+        'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6NiwiZW1haWwiOiJlbHZpc3J1Z2FtYmFAZ21haWwuY29tIiwicm9sZSI6Im1hbmFnZXIiLCJpYXQiOjE1ODI1NzA3MjF9.HisvoHo0t8cmWvpGokttlXtV07QOqUBB8dq4vqQ-s14'
```

Comment:
Why are we hardcoding this token?
Can't you generate one by registering a user?

What will happen to your test when this token expires?

Reply:
let me fix it